integration/integration_test.go

```diff
@@ -318,12 +425,10 @@ func waitForDeployment(ctx context.Context, name string, revision, timeout int)
 	return fmt.Errorf("%s didn't rollout after 30 seconds", name)
 }
 
-func getContent() (string, error) {
-	endpoint := "http://localhost:8080/index.html"
+func getContent(endpoint string) (string, error) {
 	retries := 0
-
 	t := time.NewTicker(1 * time.Second)
-	for i := 0; i < 60; i++ {
+	for i := 0; i < 120; i++ {
```

Comment:
Two minutes to get the content is a lot no?  Maybe pass this as a param so the `up` test fails sooner?